contracts/bridge/src/truth/darwinia/DarwiniaLightClient.sol

```diff
@@ -285,7 +285,7 @@ contract DarwiniaLightClient is ILightClient, Bitfield, BEEFYCommitmentScheme, B
          * @dev Check that the bitfield actually contains enough claims to be succesful, ie, >= 2/3
          */
         require(
-            countSetBits(validatorClaimsBitfield) >= threshold(),
+            countSetBits(validatorClaimsBitfield) >= (authoritySetLen * 2) / 3,
```

Comment:
I'm not sure here should be `>=` or `>`? Could you confirm?

I remember it was `>`. @hujw77

Reply:
https://github.com/paritytech/substrate/blob/f6679ddd31f68e9f1578064d58a246130c27026b/client/beefy/src/round.rs#L52
`authoritySetLen - (authoritySetLen - 1) / 3`

Reply:
I see the `threshold` at several locations. Maybe here is the right thing.